### PR TITLE
Add minimal suggestions on how to write and contribute frontends.

### PR DESCRIPTION
**Pull request by mobinmob**

### CODE REVIEW

File: Suggestions_for_services.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Suggestions for Services
+
+This document contains some suggestions on how to build service fronted service
```
Comment (teldra) [inline Suggestions_for_services.md:3]: > service fronted service

maybe withou first `service`

File: Suggestions_for_services.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+files for this repository and how to contribute them. They do not cover what the
+[official documentation](https://web.obarun.org/software/66/frontend.html) does,
+but are common sense rules for frontend building.
+
+There is not one true way to build frontends as the format is quite flexible and
+different policies and preferences will result in different practices.
+
+1. @execute scripts should be written in execline or sh. The former is preffered
```
Comment (teldra) [inline Suggestions_for_services.md:11]: preferred

File: Suggestions_for_services.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+but not mandatory. When somebody writes in sh, they must take  [the differences
+expressed in the documentation](https://web.obarun.org/software/66/frontend.html#A%20word%20about%20the%20@execute%20key) into account.
+2. @execute should contain the neccesary setup and **only** the minimum command
```
Comment (teldra) [inline Suggestions_for_services.md:14]: necessary

File: Suggestions_for_services.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+switches for the service to run (e.g. to keep it in the foreground).
+3. Every other switch should be under [environment] in *cmd_args* or other proper
+variables. 
+4. *cmd_args* replaces  *$OPTS* in scripts based on runit services.
```
Comment (teldra) [inline Suggestions_for_services.md:18]: > replaces  *$OPTS*

one whitespace to much?

File: Suggestions_for_services.md (hunk 1 above)
Comment (teldra) [reply Suggestions_for_services.md:3]: ah, frontend needs an `n`

File: Suggestions_for_services.md (hunk 4 above)
Comment (mobinmob) [reply Suggestions_for_services.md]: Thanks for the review !

**Pull request merged**